app/src/components/staking/PageValidators.vue

```diff
@@ -242,7 +242,6 @@ export default {
 .show-mobile-sorting {
   display: none;
   cursor: pointer;
-  display: none;
```

Comment:
?

Reply:
Yeah, this is just a lint thing that I have fixed in all my open PRs now